src/tests/integration/omneo/rewards-definition/util.ts

```diff
@@ -0,0 +1,4 @@
+import randomString from '../../../lib/string/random'
+
+export const getName = () => { return `sdk_unit_test_reward_definition_name_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
```

Comment:
could this instead be a generic function that can be used for all tests?

A function that accepts a string arg "name" 

then you call it with `getName("sdk_unit_test_reward_definition_name")`

Reply:
changed.